stable/influxdb/README.md

```diff
@@ -65,6 +65,39 @@ $ helm install --name my-release -f values.yaml stable/influxdb
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+### Enterprise
+[InfluxDB Enterprise](https://www.influxdata.com/products/influxdb-enterprise/) is a hardened version of the open source core InfluxDB that includes additional closed source features designed for production workloads, featuring high availability and horizontal scaling. InfluxDB Enterprise features require a InfluxDB Enterprise license.
+
+#### Configuration
+To enable InfluxDB Enterprise, set the following keys and values in a values file provided to Helm.
```

Comment:
Feel free to provide a sample `enterprise-values.yaml` file as well!

Reply:
Is that preferred? I tend to lean away from multiple values files, as the ordering is non-intuitive and easy to break (and I'm unaware if merging `.env` will do what I want). 

Everything in `.enterprise` could be kept separate, but seeing as how that part is a neat and contained, I think it makes sense to keep colocated.